lib/airrecord/table.rb

```diff
@@ -25,21 +25,30 @@ class << self
       def deprecate_symbols
         warn Kernel.caller.first + ": warning: Using symbols with airrecord is deprecated."
       end
-      
+
       def client
         @@clients ||= {}
         @@clients[api_key] ||= Client.new(api_key)
       end
 
-      def has_many(name, options)
-        @associations ||= []
-        @associations << {
-          field: name.to_sym, # todo: deprecate_symbols
-        }.merge(options)
+      def has_many(method_name, options)
+        # define an assocation getter
+        define_method(method_name.to_sym) do
+          # Get association ids in reverse order, because Airtable’s UI and API
+          # sort associations in opposite directions. We want to match the UI.
+          ids = (self[options.fetch(:column)] || []).reverse
+          table = Kernel.const_get(options.fetch(:class))
+          options[:single] ? table.find(ids.first) : table.find_many(ids)
```

Comment:
Let's create a `has_one` as well so you can express things properly? It's cool to keep `:single` though to re-use the logic here 👍

Reply:
Because Airtable always handles associations as an array, even when they're limited to a single record, I ended up just aliasing `belongs_to` and `has_one`.

I think you're right that both methods should be available, so that when the time comes to move an app from Airtable to a database, the modeling is already expressed correctly.